package.json

```diff
@@ -14,12 +14,13 @@
     "publish": "npm publish --access=public -ws"
```

Comment:
I'd suggest adding a setup script that runs install and allow-scripts in a sequence.

Reply:
```suggestion
    "publish": "npm publish --access=public -ws",
    "setup": "npm install && npm run allow-scripts"
```

like this?

Reply:
Or npm ci depending on your intentions